src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/General/MaintenanceType.php

```diff
@@ -44,24 +45,37 @@ class MaintenanceType extends TranslatorAwareType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('enable_shop', SwitchType::class, array(
-                'required' => true,
-            ))
-            ->add('maintenance_ip', IpAddressType::class, array(
-                'required' => false,
-                'attr' => array(
-                    'class' => 'col-md-5',
-                ),
-            ))
-            ->add('maintenance_text', TranslateType::class, array(
-                'type' => FormattedTextareaType::class,
-                'options' => array(
+            ->add(
+                'enable_shop',
+                SwitchType::class,
+                [
+                    'required' => true,
+                ]
+            )
+            ->add(
+                'maintenance_ip',
+                IpAddressType::class,
+                [
                     'required' => false,
-                ),
-                'locales' => $this->locales,
-                'hideTabs' => false,
-                'required' => true,
-            ))
+                    'empty_data' => '',
```

Comment:
Actual fix

Reply:
My bad I forgot to add something in my description.